src/screen/Council/MotionsScreen.tsx

```diff
@@ -16,24 +16,29 @@ import SafeView, {noTopEdges} from 'presentational/SafeView';
 import {ProposalInfo} from 'presentational/ProposalInfo';
 import {useMotions} from 'src/api/hooks/useMotions';
 import {standardPadding} from 'src/styles';
+import LoadingView from 'presentational/LoadingView';
 
 export function MotionsScreen() {
   const {data, refetch, isLoading} = useMotions();
 
   return (
     <SafeView edges={noTopEdges}>
-      <FlatList
-        refreshing={isLoading}
-        onRefresh={refetch}
-        style={styles.flatList}
-        data={data}
-        renderItem={({item}) => {
-          return <Motion item={item} />;
-        }}
-        ItemSeparatorComponent={Divider}
-        keyExtractor={(item) => item.hash.toHex()}
-        ListEmptyComponent={EmptyView}
-      />
+      {isLoading ? (
+        <LoadingView />
+      ) : (
+        <FlatList
+          refreshing={isLoading}
```

Comment:
This can be removed

Reply:
I changed it with `isFetching` to show the loading indicator on swipe down